app/hotels/src/gallery/PhotosStripe.js

```diff
@@ -14,10 +14,28 @@ type Props = {|
   onClose: () => void,
 |};
 
-export default class PhotosStripe extends React.Component<Props> {
+type State = {|
+  key: number,
+|};
+
+/* 
+  Need to trigger a re-render on android to show photo
+  https://github.com/leecade/react-native-swiper/issues/227
+*/
+export default class PhotosStripe extends React.Component<Props, State> {
+  state = {
+    key: Math.random(),
+  };
+
+  componentDidMount = () => {
+    if (Platform.OS === 'android') {
+      setTimeout(() => this.setState({ key: Math.random() }), 200);
```

Comment:
@tbergq Is this solution sufficient? Do you know why 200 and not something else?

Reply:
I have tested it, and the photo always shows. I guess the main idea here is to trigger re-render. Could be that some other value lower than 200ms also will do the trick. 

Do you want me to investigate that further?

Reply:
Yes please, give it few more minutes to understand this. For example - does it work when the image is not in the cache (i.e. we need to wait longer for the download)? Does it matter?

Reply:
removing the setState call from timeout makes the opening of the modal laggy. Having it in a timeout without any delay (making it execute async) prevents the laggy behaviour and renders the image.